app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,107 +1,56 @@
-  const [showDeleteAlert, setShowDeleteAlert] = useState(false);
+  const [showMenu, setShowMenu] = useState(false);
   const [searchTerm, setSearchTerm] = useState("");
-  const [selectedNoteIds, setSelectedNoteIds] = useState([]);
-  const [notes, setNotes] = useState([]);
-
-  useEffect(() => {
-    fetchNotes();
-  }, []);
-
-  const fetchNotes = async () => {
-    try {
-      setLoading(true);
-      const {
-        data: { notes },
-      } = await notesApi.fetch();
-      setNotes(notes);
-    } catch (error) {
-      logger.error(error);
-    } finally {
-      setLoading(false);
-    }
-  };
-
-  if (loading) {
-    return <PageLoader />;
-  }
 
   return (
-    <Container>
-      <Header
-        title="Notes"
-        actionBlock={
-          <Button
-            icon="ri-add-line"
-            label="Add new note"
-            size="small"
-            onClick={() => setShowNewNotePane(true)}
-          />
-        }
-        searchProps={{
-          value: searchTerm,
-          onChange: e => setSearchTerm(e.target.value),
-        }}
-      />
-      {notes.length ? (
-        <>
-          <SubHeader
-            rightActionBlock={
-              <Button
-                disabled={!selectedNoteIds.length}
-                icon={Delete}
-                label="Delete"
-                size="small"
-                onClick={() => setShowDeleteAlert(true)}
-              />
-            }
-          />
-          <Table
-            fetchNotes={fetchNotes}
-            notes={notes}
-            selectedNoteIds={selectedNoteIds}
-            setSelectedNoteIds={setSelectedNoteIds}
-          />
-        </>
-      ) : (
-        <EmptyState
-          image={EmptyNotesListImage}
-          primaryAction={() => setShowNewNotePane(true)}
-          primaryActionLabel="Add new note"
-          subtitle="Add your notes to send customized emails to them."
-          title="Looks like you don't have any notes!"
+    <>
+      <Menu showMenu={showMenu} />
+      <Container>
+        <Header
+          menuBarToggle={() => setShowMenu(!showMenu)}
+          title="All Notes"
+          actionBlock={
+            <Button
+              icon="ri-add-line"
+              label="Add new note"
+              size="small"
+              onClick={() => {}}
+            />
+          }
+          searchProps={{
+            value: searchTerm,
+            onChange: e => setSearchTerm(e.target.value),
+          }}
         />
-      )}
-      <NewNotePane
-        fetchNotes={fetchNotes}
-        setShowPane={setShowNewNotePane}
-        showPane={showNewNotePane}
-      />
-      {showDeleteAlert && (
-        <DeleteAlert
-          refetch={fetchNotes}
-          selectedNoteIds={selectedNoteIds}
-          setSelectedNoteIds={setSelectedNoteIds}
-          onClose={() => setShowDeleteAlert(false)}
-        />
-      )}
-    </Container>
+        {notesData.length ? (
+          <>
```

Comment:
Why do we need this fragment here?

Reply:
First, let me show you three scenarios.

Case 1: Using div tag.
`<div>
          {NOTES_CARD_DATA.map(note => (
            <Note key={"id"} note={note} />
          ))}
</div>`
![63E7F9A0-801D-45D0-914E-9D932F4FD257_1_201_a](https://user-images.githubusercontent.com/53861843/191898444-cf4cf5ad-d573-48ec-8ca7-5e2a71b58b39.jpeg)

Case 2: Using React Fragment.
`<>
          {NOTES_CARD_DATA.map(note => (
            <Note key={"id"} note={note} />
          ))}
</>`
![377A4360-623E-492A-9822-0E9F801A06C6_1_201_a](https://user-images.githubusercontent.com/53861843/191898382-30065cff-826f-49d4-933c-e52b83cee59b.jpeg)

Case 3: Without any tag for wrapping.
<img width="864" alt="Screenshot 2022-09-23 at 11 06 07 AM" src="https://user-images.githubusercontent.com/53861843/191898483-e1c6d4c2-4635-43fb-9b34-fde099465d98.png">
For my convenience I have put the screenshot from VScode the same error is shown in terminal also.

From my understanding,
In the JSX expression returning multiple items, I should wrap it inside a tag.
Then, I have two options to wrap it inside a <></> or <div></div>.

According to  React Best practices, https://academy.bigbinary.com/learn-react/pages/using-appropriate-tags, I think its okay to use the React Fragment.

Also I don't understand why the output is different for <></> or <div></div> tags.

Can you please correct me if I am doing anything wrong here?

Reply:
I believe it's because once a div is present it acts as a container and we will need to specify it occupy the full width of parent. But Fragments are not adding any real nodes in the DOM.

Refer: https://blog.logrocket.com/understanding-react-fragments/

You can keep the Fragment. I just wanted to make sure that there's a valid reason for adding the same.